client/src/components/endpoint-controls.js

```diff
@@ -28,11 +28,11 @@ class EndpointControls extends React.Component {
         return (<header>
             <div className="container">
                 <div className="endpoint-controls">
-                    <h1>This Is Your Endpoint</h1>
+                    <h1>This Is Your Token</h1>
 
                     <div className="row">
                         <div className="column column-60">
-                            <input ref="urlInput" type="text" value={this.state.loading ? 'Loading...' : this.state.link} readOnly />
+                            <input ref="urlInput" type="text" value={this.state.loading ? 'Loading...' : this.state.token} readOnly />
```

Comment:
Awesome, however I still want the full link available in the website, so let's change the value of the input to be `value={this.state.loading ? 'Loading...' : `${location.protocol}//${location.hostname}/${this.state.token}`}` or something like that